packages/core/test/helpers/constants.ts

```diff
@@ -38,6 +39,10 @@ export const predefinedNodesTypes: INodeTypeData = {
 		type: new ManualTrigger(),
 		sourcePath: '',
 	},
+	'n8n-nodes-base.splitInBatches': {
+		type: new SplitInBatches(),
+		sourcePath: '',
+	},
```

Comment:
This node has special handling and I want to test this with an integration test in `packages/core/test/WorkflowExecute.test.ts`. To do this I need the full node to be available, because the function `runPartialWorkflow2` expects a `Workflow` which uses the node types.